### DISCUSSION TITLE
Suggestion: Killer cells only damage cells not entire organisms

### DISCUSSION THREAD

**Discussion by Phrenotopia** [MaxRobinsonTheGreat/EvolutionSimulatorV2#95] [Ideas and Suggestions]
Just a thought I had and not sure how tricky this will be to implement, but I was thinking that it would be a tad more realistic and interesting, if killer cells only damage those cells they're adjacent to and not the entire organism, which those cells belong to. I'm guessing this will be taken down to genetic expression, and perhaps this is also where regeneration can take place at some cost.

**Comment by Nogardagem** [comment]
I had a similar idea, where any cell hit by a predator cell would be deleted, so the organism would always be alive after being hurt unless it lost all of its cells, and it could also die from just being injured.

**Reply by Phrenotopia** [reply]
Exactly!

**Comment by theomgdev** [comment]
It will probably unlock the evolution of over-grown organisms mostly made of defense cells(armor, kill)
